backend/app/assets/javascripts/spree/backend/orders/models.js

```diff
@@ -0,0 +1,48 @@
+//= require spree/backend/routes
+
+Spree.Order || (Spree.Order = {});
+
+
+Spree.Order.LineItemModel = Backbone.Model.extend({
+  initialize: function(options) {
+    this.order = this.collection.parent;
+  }
+})
+
+Spree.Order.LineItemCollection = Backbone.Collection.extend({
+  model: Spree.Order.LineItemModel,
+
+  url: function () {
+    return this.parent.url() + "/line_items";
+  },
+})
+
+Spree.Order.OrderModel = Backbone.Model.extend({
+  urlRoot: Spree.routes.orders_api,
+  idAttribute: "number",
+
+  relations: {
+    "line_items": Spree.Order.LineItemCollection,
+    "shipments": Backbone.Collection
+  },
+
+  advance: function(opts) {
+    var options = {
+      url: Spree.routes.checkouts_api + "/" + this.id + "/advance",
+      type: 'PUT',
+    };
+    _.extend(options, opts);
+    return this.fetch(options)
+  }
+});
+
+Spree.Order.OrderModel.fetch = function(number, opts) {
+  var options = (opts || {});
+  var model = new Spree.Order.OrderModel({
+    number: number,
+    line_items: [],
+    shipments: []
+  });
+  model.fetch(options);
+  return model;
+}
```

Comment:
I agree. Going to split these into separate files and also move to a different namespace, `Spree.Models.Order`, which will also make more sense as we add more models (like `Spree.Models.Variant`)